### DISCUSSION TITLE
Search bar on lenses

### DISCUSSION THREAD

**Discussion by sanderbaas** [laravel/nova-issues#4752] [Ideas & Feature Requests] [answered]
Is it possible to add the search bar we have on a default resource page to lens pages?

I found an old feature requests (#1750) about implementing the search bar for lenses. @davidhemphill says there that the user defines the query ahead of time, but as @MannikJ also mentioned, why does this prevent us from using an extra search instruction?

The users of the app I am making are confused about the search bar missing from some pages and I can not really explain it to them.

**Comment by crynobone** [comment, marked as answer]
This has been added on v4.21.0: https://nova.laravel.com/releases/4.21.0

API is similar to Resource.
